events/README.md

```diff
@@ -247,6 +248,24 @@ Type(s) of lane used or blocked `vehicle_blocked_lane_types` by the vehicle perf
 
 [Top][toc]
 
+### Payment Type
+
+Strings used to indicate how a curb user paid for a curb event.
+
+| Name              | Description                                            |
+| ----------------- | ------------------------------------------------------ |
+| `cash`            | Bills or coins at a meter. |
+| `credit_card`     | Visa, Mastercard, etc at a meter. |
+| `membership_card` | A card used at a meter to pay via a corporate membership or loyalta program, etc. |
+| `mobile`          | Paid via a mobile app. |
```

Comment:
`mobile` could potentially be ambiguous, since payment with a mobile app could be done with credit/debit cards, but also with digital wallets, prepaid accounts, permits, etc.